module4/labs/Week3/MaintainingState.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Maintaining State Lab
+---
+
+In today's lab, we are going to build on the [Start Repo](https://github.com/turingschool-examples/GoodBooksMvcStarter) from the lesson.
```

Comment:
Starter Repo, not Start Repo

I like that you used the same repo! Less time wasted mucking through starting up a new app.